src/math/random.js

```diff
@@ -149,59 +165,47 @@ p5.prototype.random = function(min, max) {
 };
 
 /**
+ * Returns a random number fitting a Gaussian, or normal, distribution.
```

Comment:
Maybe adding "...which looks like a bell curve" would help people picture the distribution in their heads

Reply:
Good suggestion!